File: WorldFeed/Server/src/WorldFeed.Common/DTO/TweetFromSearchMetadata.cs
namespace WorldFeed.Common.DTO
{
    using Newtonsoft.Json;

    using WorldFeed.Common.Public.Models.Interfaces.DTO;

    public class TweetFromSearchMetadata : ITweetFromSearchMetadata
    {
        [JsonProperty("result_type")]
        public string ResultType { get; private set; }

        [JsonProperty("iso_language_code")]
        public string IsoLanguageCode { get; private set; }
    }
}

File: WorldFeed/Client/src/app/core/Core/DTO/TweetFromSearchMetadata.ts
import {ITweetFromSearchMetadata} from "../../Public/Models/Interfaces/DTO/ITweetFromSearchMetadata";

export class TweetFromSearchMetadata implements ITweetFromSearchMetadata {
  // [JsonProperty("result_type")]
  public resultType: string;

  // [JsonProperty("iso_language_code")]
  public isoLanguageCode: string;
}
